plugins/importexport/csv/version.xml

```diff
@@ -13,6 +13,6 @@
 <version>
 	<application>csv</application>
 	<type>plugins.importexport</type>
-	<release>1.0.0.0</release>
+	<release>2.0.0.0</release>
```

Comment:
If this isn't a complete rewrite, I'd suggest just bumping the minor release number to e.g. `1.0.1.0`.